sample_consensus/include/pcl/sample_consensus/impl/ransac.hpp

```diff
@@ -55,77 +58,148 @@ pcl::RandomSampleConsensus<PointT>::computeModel (int)
   }
 
   iterations_ = 0;
-  int n_best_inliers_count = -INT_MAX;
-  double k = 1.0;
+  std::size_t n_best_inliers_count = 0;
+  double k = INFINITY;
```

Comment:
This is INFINITY: https://en.cppreference.com/w/cpp/numeric/math/INFINITY
I guess `max()` would work equally well.
@taketwo what is your opinion on this?

Reply:
That's a C macro (accidental narrow, bad naming, scopeless) so, C++ numeric_limits are preferred

Reply:
> If the implementation does not support floating-point infinities, the macro INFINITY expands to a positive value that is guaranteed to overflow a float at compile time, and the use of this macro generates a *compiler warning*. 

I guess it's hard to find implementations without floating-point infinities in the wild. Still, I'd say `max` is a safer bet since it's always supported.